### ISSUE TITLE
Windows relying on retrieveInProgressSpectatorGameInfo no longer display summoner names

### ISSUE THREAD

**Issue by jstock78** [SightstoneOfficial/LegendaryClient#417]
It looks like retrieveInProgressSpectatorGameInfo has changed and no longer returns summoner names.    This causes problems with anywhere the Legendary Client displays live stats, such as the InGame tab.  I tried to research the problem but the account id that is returned doesn't seem to correlate to anything I'm familiar with.  Any ideas?

**Comment by eddy5641** [comment]
I am unable to check what has changed
Do you have an account id that is returned and stats about that account?
(Maybe the acc name acc id)
if not i can check later on

**Comment by jstock78** [comment]
Here are the easiest steps I found to reproduce:
1) Start the LC and logged into NA with my account.
2) Make note of a few summoner names from a featured match (at least 2 from the same match).  In this example I see "Wardrium", "GodPiglet", "Nightblue3", "Kurisuchan", and "Waker" all playing in the same match on the same team.
3) Look one of their profiles up in the client (I looked up "Waker" in this example).  The Ingame tab will appear, but all profile names will be empty except for Waker.  Same thing in GameScouter.

Here is what's returned from the RiotCalls.RetrieveInProgressSpectatorGameInfo() method for "Wardrium"

Wardrium
RetrieveInProgressSpectatorGameInfo
AccountId   2399982326.0    double
SummonerId  0.0 double
SummonerInternalName    "summoner2399982326"    string
SummonerName    ""  string

Here is what GetSummonerByName returns for "Wardrium":
AcctId  39091818.0  double
InternalName    null    string
Name    "Wardrium"  string
ProfileIconId   26  int
+RevisionDate   {8/6/2015 4:26:59 PM}   System.DateTime
RevisionId  0.0 double
SummonerId  24694433.0  double
SummonerLevel   30.0    double

As far as I can tell the account id returned by RetrieveInProgressSpectatorGameInfo is random, but there might be some pattern to it.
